### ISSUE TITLE
error

### ISSUE THREAD

**Issue by Lornot** [avocode/avocode#390]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.4.1-9839e14
Source UID: 25256
```
TypeError: Cannot read property 'x' of null
  at e.calculateBounds (C:\Users\Lenovo\AppData\Local\avocode\app-2.4.1\resources\app.asar\node_modules\@avocode\select-tool\lib\select-tool.js:1:8095)
  at e.mousemoveDrag (C:\Users\Lenovo\AppData\Local\avocode\app-2.4.1\resources\app.asar\node_modules\@avocode\select-tool\lib\select-tool.js:1:4971)
  at C:\Users\Lenovo\AppData\Local\avocode\app-2.4.1\resources\app.asar\node_modules\@avocode\select-tool\lib\select-tool.js:1:240
  at t.module.exports.Emitter.emit (C:\Users\Lenovo\AppData\Local\avocode\app-2.4.1\resources\app.asar\node_modules\emissary\lib\emitter.js:118:15)
  at t.mousewheel (C:\Users\Lenovo\AppData\Local\avocode\app-2.4.1\resources\app.asar\node_modules\@avocode\core\lib\stage.js:1:13091)
  at HTMLCanvasElement.<anonymous> (C:\Users\Lenovo\AppData\Local\avocode\app-2.4.1\resources\app.asar\node_modules\@avocode\core\lib\stage.js:1:438)

```

**Comment by petrbrzek** [comment]
Hello, we’ve just released Avocode 2.4.3 that should fix the issue. Let us know if the problem persists.

**Comment by Lornot** [comment]
wow, thanks a lot

2015-09-23 17:15 GMT+03:00 Petr Brzek <notifications@github.com>:
